automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/user/RegistrationSpec.scala

```diff
@@ -16,9 +16,9 @@ import org.scalatest.{BeforeAndAfter, FreeSpec, Matchers}
 /**
   * Tests for new user registration scenarios.
   */
-class RegistrationSpec extends FreeSpec with FailTestRetryable with BeforeAndAfter with Matchers with WebBrowserSpec with UserFixtures {
+class RegistrationSpec extends FreeSpec with BeforeAndAfter with Matchers with WebBrowserSpec with UserFixtures with FailTestRetryable {
 
-  val testUser: Credentials = FireCloudConfig.Users.temp // TODO: pull from user pool and fetch correct subject ID
+  val testUser: Credentials = FireCloudConfig.Users.Temps.getUserCredential("luna")
```

Comment:
fixed user so that xforwarding is done once in alpha. repeating xforwarding for every temp user is too much on maintenance.